quirc/examples/esp-eye-qrcode/main/app_main.c

```diff
@@ -0,0 +1,85 @@
+#include <stdio.h>
+#include <stdint.h>
+#include <string.h>
+#include "esp_log.h"
+#include "esp_system.h"
+#include "freertos/FreeRTOS.h"
+#include "freertos/task.h"
+#include "app_peripherals.h"
+#include "quirc.h"
+
+static const char *TAG = "APP_CODE_SCANNER";
+
+static void decode_task()
```

Comment:
FreeRTOS task function definition should look like this:
```suggestion
static void decode_task(void* arg)
```